lib/src/validation_result.dart

```diff
@@ -90,28 +92,41 @@ class ValidationResult {
       map['resources'] = context.resources;
     }
 
-    map['hasAnimations'] = root.animations.isNotEmpty;
-    map['hasMaterials'] = root.materials.isNotEmpty;
+    map['animationCount'] = root.animations.length;
+    map['materialCount'] = root.materials.length;
     map['hasMorphTargets'] = root.meshes.any((mesh) =>
         mesh.primitives != null &&
         mesh.primitives.any((primitive) => primitive.targets != null));
     map['hasSkins'] = root.skins.isNotEmpty;
     map['hasTextures'] = root.textures.isNotEmpty;
     map['hasDefaultScene'] = root.scene != null;
 
-    var primitivesCount = 0;
-    var maxAttributesUsed = 0;
+    var drawCallCount = 0;
+    var maxAttributes = 0;
+    var maxUVs = 0;
+    var maxInfluences = 0;
+    var totalVertexCount = 0;
+    var totalTriangleCount = 0;
     for (final mesh in root.meshes) {
       if (mesh.primitives != null) {
-        primitivesCount += mesh.primitives.length;
+        drawCallCount += mesh.primitives.length;
         for (final primitive in mesh.primitives) {
-          maxAttributesUsed =
-              max(maxAttributesUsed, primitive.attributes.length);
+          if (primitive.vertexCount != -1) {
+            totalVertexCount += primitive.vertexCount;
+          }
+          totalTriangleCount += primitive.trianglesCount;
+          maxAttributes = max(maxAttributes, primitive.attributes.length);
+          maxUVs = max(maxUVs, primitive.texcoordCount);
+          maxInfluences = max(maxInfluences, primitive.jointsCount * 4);
         }
       }
     }
-    map['primitivesCount'] = primitivesCount;
-    map['maxAttributesUsed'] = maxAttributesUsed;
+    map['drawCallCount'] = drawCallCount;
+    map['totalVertexCount'] = totalVertexCount;
+    map['totalTriangleCount'] = totalTriangleCount;
+    map['maxUVs'] = maxUVs;
+    map['maxInfluences'] = maxInfluences;
+    map['maxAttributes'] = maxAttributes;
```

Comment:
Great to have this info reported, thanks! This will be helpful for my viewer as well. 👍